XRTK-Core/Packages/com.xrtk.core/Runtime/Definitions/Controllers/Hands/RecordedHandJoints.cs

```diff
@@ -10,14 +10,19 @@ namespace XRTK.Definitions.Controllers.Hands
     /// Unity's <see cref="JsonUtility"/> "currently" (stated in 2011) does not support top-level arrays.
     /// But hey, it's on their "future" road-map. That's why we need a wrapper definition around or items.
     /// </summary>
-    /// <remarks>The <see cref="JsonUtility"/> also does not support properties, so we gotta use fields. Oh and it also
-    /// does not support auto mapping of JSON naming "items" to C# naming "Items".</remarks>
```

Comment:
Actually it does support properties as long as the fields are serialized. Standard naming should be okay in this case. It's when we don't have control of the data structure that we might run into problems here.